WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java

```diff
@@ -1558,18 +1584,91 @@ public void onItemClick(AdapterView<?> parent, View view, int position, long id)
                 listPopup.dismiss();
                 switch ((int) id) {
                     case ReaderMenuAdapter.ITEM_FOLLOW:
+                        onFollowTapped(getView(), post.getBlogName(), post.blogId);
+                        toggleFollowStatusForPost(post);
+                        break;
                     case ReaderMenuAdapter.ITEM_UNFOLLOW:
+                        onFollowingTapped();
                         toggleFollowStatusForPost(post);
                         break;
                     case ReaderMenuAdapter.ITEM_BLOCK:
                         blockBlogForPost(post);
                         break;
+                    case ReaderMenuAdapter.ITEM_NOTIFICATIONS_OFF:
+                        AnalyticsUtils.trackWithSiteId(Stat.FOLLOWED_BLOG_NOTIFICATIONS_READER_MENU_OFF, post.blogId);
+                        ReaderBlogTable.setNotificationsEnabledByBlogId(post.blogId, false);
+                        updateSubscription(SubscriptionAction.DELETE, post.blogId);
+                        break;
+                    case ReaderMenuAdapter.ITEM_NOTIFICATIONS_ON:
+                        AnalyticsUtils.trackWithSiteId(Stat.FOLLOWED_BLOG_NOTIFICATIONS_READER_MENU_ON, post.blogId);
+                        ReaderBlogTable.setNotificationsEnabledByBlogId(post.blogId, true);
+                        updateSubscription(SubscriptionAction.NEW, post.blogId);
+                        break;
+                    case ReaderMenuAdapter.ITEM_SHARE:
+                        AnalyticsUtils.trackWithSiteId(Stat.SHARED_ITEM_READER, post.blogId);
+                        sharePost(post);
+                        break;
                 }
             }
         });
         listPopup.show();
     }
 
+    @Override
+    public void onFollowTapped(View view, String blogName, final long blogId) {
+        mDispatcher.dispatch(AccountActionBuilder.newFetchSubscriptionsAction());
+
+        Snackbar.make(view, Html.fromHtml(getString(R.string.reader_followed_blog_notifications,
+                        "<b>", blogName, "</b>")), Snackbar.LENGTH_LONG)
```

Comment:
When settings a duration of actionable Snackbars, I suggest using AccessibilityUtils.getSnackbarDuration. It designed to provide a longer delay for users who have accessibility services enabled.

Reply:
I didn't know about `AccessibilityUtils.getSnackbarDuration`.  Thanks for the tip!  I updated the durations in 25d0f71.